lib/Twig/Environment.php

```diff
@@ -1402,6 +1402,24 @@ public function computeAlternatives($name, $items)
         return Twig_Error_Syntax::computeAlternatives($name, $items);
     }
 
+    public function addUnaryOperator(array $operator)
+    {
+        if ($this->extensionInitialized) {
+            throw new LogicException('Unable to add unary operator as extensions have already been initialized.');
+        }
+
+        $this->unaryOperators = array_merge($this->unaryOperators, $operator);
```

Comment:
yeah, because it was not possible to register them without extension previously. Your PR should precisely be adding them as it adds the feature.